### PR TITLE
feat: Add Simple Array Sum HackerRank Js Coding Challenge

### PR DESCRIPTION
**Pull request by moshoodabdullahi**
# This PR adds the simple array sum hackerRank js coding challenge

### CODE REVIEW

File: hackerRank/simple-array-sum/simple-array-sum.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const simpleArraySum = (ar) => {
+  let sumOfArray = 0;
+  for (let i = 0; i < ar.length; i += 1) {
```
Comment (Yinkusadel) [inline hackerRank/simple-array-sum/simple-array-sum.js]: Lets refactor this to use the for each loop

File: hackerRank/simple-array-sum/simple-array-sum.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const simpleArraySum = require('./simple-array-sum');
+const testCaseFileOne = require('./testCaseFileOne');
+
+test('adds 1 + 2 + 3 + 4 + 10 + 11  to equal 31', () => {
```
Comment (Yinkusadel) [inline hackerRank/simple-array-sum/simple-array-sum.test.js]: Lets update this description to something better "sums an array for integers" maybe,but something more descriptive

**Review by Yinkusadel — CHANGES_REQUESTED** [review]

File: hackerRank/simple-array-sum/simple-array-sum.test.js (hunk 2 above)
Comment (moshoodabdullahi) [reply hackerRank/simple-array-sum/simple-array-sum.test.js]: Done

**Review by Yinkusadel — APPROVED** [review]
nice

**Review by teckcipher — APPROVED** [review]

**Pull request merged**